introduction.md

```diff
@@ -5,4 +5,7 @@ This is the introduction
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?